tensorflow_lite_support/ios/test/task/vision/image_classifier/TFLImageClassifierTests.m

```diff
@@ -37,7 +38,9 @@ - (void)testSuccessfullImageInferenceOnMLImageWithUIImage {
   [TFLImageClassifier imageClassifierWithModelPath:self.modelPath error:nil];
   XCTAssertNotNil(imageClassifier);
   
-  UIImage *image = [UIImage imageNamed:@"burger_crop"];
+  NSString *imageName = [[NSBundle bundleForClass:[self class]] pathForResource:@"burger_crop"
```

Comment:
nit: 
* This should be `imagePath` rather than `imageName` 
* As you access `[[NSBundle bundleForClass:[self class]] pathForResource` quite often, maybe you can create a util method like pathFromBundle:(NSString *)filename

Reply:
Encapsulated the image initialisation in a method since all that code is repeated